### ISSUE TITLE
html template using byTags sidebarOrganization does not look through all the tags

### ISSUE THREAD

**Issue by jonaslagoni** [asyncapi/generator#148]
While solving #141 I saw that only the first tag are checked:
https://github.com/asyncapi/generator/blob/f1c196a8cb12a667d4dbae5b01af6b538ca184d9/templates/html/.partials/sidebar.html#L47
Meaning that if I define operations like:
```json
...
"tags": [{
	"name": "user1",
	"description": "User-related messages"
},
{
	"name": "user",
	"description": "User-related messages"
}
],
...
```
and the tags like:
```json
...
"tags": [
{
	"name": "user",
	"description": "User-related messages"
}
],
...
```
the operation would not be listed under the tags, guess this is not intended?

**Comment by fmvilas** [comment]
You're right, this is an issue.

**Comment by jonaslagoni** [comment]
This was not yet implemented in PR #147 gonna make fix this bug now.
